### PR TITLE
Adapt to new C3I rules regarding bumps

### PR DESCRIPTION
**Pull request by ericLemanissier**
Also, make the summary issue optional and configurable.
This requires the setup of a Repository variable named `ISSUE_NUMBER` in https://github.com/prince-chrismc/conan-center-index-pending-review/settings/variables/actions

### CODE REVIEW

**Comment by ericLemanissier** [comment]
results can be viewed on  https://ericlemanissier.github.io/conan-center-index-pending-review/

File: cmd/cpr/pending_review.go [hunk 1]
```diff
@@ -106,11 +106,13 @@ func PendingReview(token string, dryRun bool, owner string, repo string) error {
 		return fmt.Errorf("problem editing web view %w", err)
 	}
 
-	_, _, err = client.Issues.Edit(context, owner, repo, 1, &github.IssueRequest{
-		Body: github.String(commentBody),
-	})
-	if err != nil {
-		return fmt.Errorf("problem editing issue %w", err)
+	if issue != 0 {
```
Comment (prince-chrismc) [inline cmd/cpr/pending_review.go:109]: I think it would be helpful to log that it's not updating the issue

File: pending_review/review.go [hunk 2]
```diff
@@ -25,11 +25,17 @@ type Reviews struct {
...
 }
 
 // IsApproved when the conditions for merging are meet as per https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md
 func (r *Reviews) IsApproved() bool {
-	return r.TeamApproval && r.ValidApprovals >= 2 && len(r.Blockers) == 0
+	var nRequiredApprovals = 2
+	if r.IsBump {
+		nRequiredApprovals = 1
+	}
+	return r.TeamApproval && r.ValidApprovals >= nRequiredApprovals && len(r.Blockers) == 0
```
Comment (prince-chrismc) [inline pending_review/review.go:38]: Do dump PR need to be accepted by a team member or  are community members counted too?

**Review by prince-chrismc — COMMENTED** [review]
I think it's worth adding a new twst for `isBump == true` with a team or community review so the logic is clear in the tests

File: pending_review/review.go (hunk 2 above)
Comment (ericLemanissier) [reply pending_review/review.go:38]: bump PRs need a team member approval cf https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md#rule-of-2-reviews

> Pull requests labelled as [Bump version](https://github.com/conan-io/conan-center-index/pulls?q=is%3Aopen+is%3Apr+label%3A%22Bump+version%22) or [Bump dependencies](https://github.com/conan-io/conan-center-index/pulls?q=is%3Aopen+is%3Apr+label%3A%22Bump+dependencies%22+) require just 1 approving review from an official reviewer. Community reviewers are not required but are still welcome.

**Comment by ericLemanissier** [comment]
Please don't merge now, I need to mock the http request in pull_reviews_test

**Comment by ericLemanissier** [comment]
@prince-chrismc this is ready for merge

File: cmd/cpr/pending_review.go [hunk 3]
```diff
@@ -106,11 +106,15 @@ func PendingReview(token string, dryRun bool, owner string, repo string) error {
 		return fmt.Errorf("problem editing web view %w", err)
 	}
 
-	_, _, err = client.Issues.Edit(context, owner, repo, 1, &github.IssueRequest{
-		Body: github.String(commentBody),
-	})
-	if err != nil {
-		return fmt.Errorf("problem editing issue %w", err)
+	if issue == 0 {
+		fmt.Printf("No issue to update\n")
```
Comment (prince-chrismc) [inline cmd/cpr/pending_review.go:110]:
```suggestion
		fmt.Println("::warning::No issue to update")
```

**Review by prince-chrismc — COMMENTED** [review]
To tiny nits

**Pull request merged**